pandas/tests/test_window.py

```diff
@@ -667,6 +667,18 @@ def test_rolling_axis(self, axis_frame):
         result = df.rolling(3, axis=axis_frame).sum()
         tm.assert_frame_equal(result, expected)
 
+    def test_count_axis(self):
```

Comment:
rename this to test_rolling_axis_count, rename the test right above to test_rolling_axis_sum